docs/src/Box.doc.js

```diff
@@ -1,24 +1,154 @@
 // @flow strict
 import React, { type Node } from 'react';
-import { Box, Button } from 'gestalt';
+import { Box } from 'gestalt';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
-import Combination from './components/Combination.js';
+import CombinationNew from './components/CombinationNew.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Box"
-    description={`In the darkest night, Box will rise to bring the light. The Lloyd has spoken.
-
-&mdash; Anon _(Winning Box Haiku, 2017)_`}
+    description="Box is a low level component that can be used to build the foundation of pretty much anything. Using Box allows you to focus on the important content, without worrying about the pixel details."
   />,
 );
 
+card(
+  <MainSection name="Examples" showHeading={false}>
+    <MainSection.Subsection
+      description={`The following examples are all built using the \`Box\` component and its various properties. `}
+    >
+      <MainSection.Card
+        title="Media Object"
+        description={`
+           The [media object](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) is a common pattern for displaying data. What's interesting about this example is the use of \`flex\` to align the items. If you try changing the size of the \`Avatar\` or the number of lines of \`Text\`, both will stay aligned because they are center aligned.
+
+           In the double-sided example we use \`flex="grow"\` to mark a flex child as something that can expand. Try removing the \`grow\` property and see what happens.
```

Comment:
This is a writer question... what's the tone of out text? 
`Try removing the \`grow\` property and see what happens.` We should discuss the tone of the Docs, is it a friendly voice that guide's you in the discovery of the Docs or just a standard Doc voice that simply describes?

I thnk it's important to get to the voice, maybe in a second round, but the sooner the less we'll have to change of wor already done.